packages/core/src/transports/blockcypher/adapters.ts

```diff
@@ -0,0 +1,14 @@
+import type { UTXO } from '../../types/transaction.js'
+import type { BlockcypherUTXO } from './blockcypher.types.js'
+
+export const blockcypherDataAdapters = {
```

Comment:
What do you think about moving this closer to implementation?

Reply:
I agree that would be more appropriate.

I had this 'adapter.ts' structure before splitting the actions into separate files.
Since the action are now in separate file it makes sense for each data transformer to be in the function file.